db/seeds.rb

```diff
@@ -39,10 +42,11 @@
 end
 
 # make an initial user for sake of the readme
-User.create(
-  email: "admin@example.com",
-  first_name: "Example",
-  last_name: "User",
-  admin: true,
-  invited: "2020-03-25 00:00:00"
-)
\ No newline at end of file
+seed_user_emails = ENV['SEED_USER_EMAILS'] || 'admin@example.com'
+seed_user_emails.split(',').each do |email|
+  User.create(
```

Comment:
Actually I’d like to use User.find_or_initialize_by(email: email.strip).update!(admin: true, invited:DateTime.now) so it won’t error if you run seed a second time.